src/dotnet/CarbonAware.DataSources.WattTime/src/WattTimeDataSource.cs

```diff
@@ -100,4 +100,11 @@ private async Task<IEnumerable<EmissionsData>> GetCarbonIntensityAsync(Location
             return result;
         }
     }
+
+    private double ConvertMoerToGramsPerKilowattHour(double value)
+    {
+        const double mwhTokwhConversionFactor = 1000.0;
```

Comment:
suggestion: move the magic number constants to the top of the file
stylistic nit: Spell them in caps with _ i.e. ```MWH_TO_KWH_CONVERSION_FACTOR```